tensorflow/python/ops/confusion_matrix.py

```diff
@@ -26,7 +26,6 @@
 from tensorflow.python.ops import control_flow_ops
 from tensorflow.python.ops import math_ops
 from tensorflow.python.ops import sparse_ops
-from tensorflow.python.util import deprecation
```

Comment:
Keep this import. It is still used below in this file.

Reply:
Updated.